### PR TITLE
Feature/tvmaze season posters

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Needs rebase after the tmdb season poster PR.

### CODE REVIEW

File: medusa/indexers/tvmaze/tvmaze_api.py [hunk 1]
```diff
@@ -301,18 +300,41 @@ def _parse_images(self, tvmaze_id):
...
+            if season not in _images['season']['original']:
+                _images['season']['original'][season] = {seasons[season].id: {}}
+            _images['season']['original'][season][seasons[season].id]['_bannerpath'] = seasons[season].image['original']
+            _images['season']['original'][season][seasons[season].id]['rating'] = 1
+
+        return _images
+
     def _parse_actors(self, tvmaze_id):
         """Parsers actors XML, from
         http://theTMDB.com/api/[APIKEY]/series/[SERIES ID]/actors.xml
```
Comment (duramato) [inline medusa/indexers/tvmaze/tvmaze_api.py]: Wrong URL?

Comment (p0psicles) [inline medusa/indexers/tvmaze/tvmaze_api.py]: It's in comments. I'm going to clean that up ltr. Need to do that for all indexers anyway.

**Comment by fernandog** [comment]
@duramato do you want me to add this commits in the ratio PR so you can test it ?

**Comment by fernandog** [comment]
done

**Pull request merged**